sw/device/lib/aes.h

```diff
@@ -14,6 +14,16 @@
  */
 typedef enum aes_op { kAesEnc = 0, kAesDec = 1 } aes_op_t;
 
+/**
+ * Supported AES block cipher modes: ECB, CBC, CTR. The hardware uses a one-hot
+ * encoding.
+ */
+typedef enum aes_mode {
```

Comment:
I have often seen this enumeration expressed more like 1<<0, 1<<1, 1<<2, etc just to really make clear the one-hot-ness.

Reply:
Done